templates/content-page.php

```diff
@@ -1,13 +1,13 @@
 <?php
 the_content();
 
-if ( ! is_page( array( 'sitemap' ) ) ) {
+if ( function_exists( 'is_page' ) && ! is_page( array( 'sitemap' ) ) && function_exists( 'is_front_page' ) && ! is_front_page() ) {
```

Comment:
Do we really need to check if the function exists? It won't work anything if doesn't exists.